$NetBSD: patch-aa,v 1.3 2020/06/21 04:09:24 mef Exp $

date: 2007-01-31 08:52:53 +0900;  author: wiz;  state: Exp;
Second try at fixing the bulk build of this package --
let the curses test always succeed.

--- configure.orig	2011-03-04 19:05:52.000000000 +0000
+++ configure
@@ -3294,6 +3294,8 @@ rm -f core *.core core.conftest.* gmon.o
 fi
 
 LIBS=$ORIG_LIBS
+curses_ok=yes
+
 if test "$curses_ok" = yes; then
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for modern curses without raw() bug" >&5
 $as_echo_n "checking for modern curses without raw() bug... " >&6; }
